cumulus/pallets/dmp-queue/Cargo.toml

```diff
@@ -23,25 +29,34 @@ xcm = { package = "staging-xcm", path = "../../../polkadot/xcm", default-feature
 cumulus-primitives-core = { path = "../../primitives/core", default-features = false }
 
 [dev-dependencies]
-sp-core = { path = "../../../substrate/primitives/core", default-features = false}
-sp-version = { path = "../../../substrate/primitives/version", default-features = false}
+sp-core = { path = "../../../substrate/primitives/core" }
+sp-tracing = { path = "../../../substrate/primitives/tracing" }
 
 [features]
 default = [ "std" ]
+
 std = [
 	"codec/std",
 	"cumulus-primitives-core/std",
+	"frame-benchmarking?/std",
 	"frame-support/std",
 	"frame-system/std",
 	"log/std",
 	"scale-info/std",
-	"sp-core/std",
 	"sp-io/std",
 	"sp-runtime/std",
 	"sp-std/std",
-	"sp-version/std",
 	"xcm/std",
 ]
+
+runtime-benchmarks = [
+	"cumulus-primitives-core/runtime-benchmarks",
+	"frame-benchmarking/runtime-benchmarks",
```

Comment:
I forgot, does this also bring in `frame-benchmarking` as a required dependency?

Reply:
WDYM? `frame-benchmarking` is optional, but will be activated by the `runtime-benchmarks` feature so it can be used in benchmarking code.